walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,137 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void everyAttendeeIsConsideredExceptOptional() {
+    // Have each person have different events. We should see two options because each person has
+    // split the restricted times. Optional person is ignored
+    //
+    // Events  :       |--A--|     |--B--|
+    // Optional: |-------------C---------------|
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+
+    Collection<Event> events = Arrays.asList(
```

Comment:
nit: let's use ImmutableLists when we can, eg: ImmutableList.of() over Arrays.asList(). Same for below.